Frontend/build/index.html

```diff
@@ -1,113 +1 @@
-                    return e.default;
-                  }
-                : function() {
-                    return e;
-                  };
-            return p.d(r, "a", r), r;
-          }),
-          (p.o = function(e, r) {
-            return Object.prototype.hasOwnProperty.call(e, r);
-          }),
-          (p.p = "/");
-        var r = (this.webpackJsonpdscwebapp = this.webpackJsonpdscwebapp || []),
-          n = r.push.bind(r);
-        (r.push = e), (r = r.slice());
-        for (var o = 0; o < r.length; o++) e(r[o]);
-        var s = n;
-        f();
-      })([]);
-    </script>
-    <script src="/static/js/2.6c054184.chunk.js"></script>
-    <script src="/static/js/main.02a170a1.chunk.js"></script>
-  </body>
-</html>
+<!doctype html><html lang="en"><head><meta charset="utf-8"/><link rel="icon" href="./favicon.ico"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="theme-color" content="#000000"/><meta name="Official website of DSC Thapar" content="Developer Student Club Thapar"/><link rel="apple-touch-icon" href="./logo192.png"/><link rel="manifest" href="./manifest.json"/><title>DSC-TIET</title><link href="./static/css/main.2c1fd7b4.chunk.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div><script>!function(e){function r(r){for(var n,p,l=r[0],a=r[1],f=r[2],c=0,s=[];c<l.length;c++)p=l[c],Object.prototype.hasOwnProperty.call(o,p)&&o[p]&&s.push(o[p][0]),o[p]=0;for(n in a)Object.prototype.hasOwnProperty.call(a,n)&&(e[n]=a[n]);for(i&&i(r);s.length;)s.shift()();return u.push.apply(u,f||[]),t()}function t(){for(var e,r=0;r<u.length;r++){for(var t=u[r],n=!0,l=1;l<t.length;l++){var a=t[l];0!==o[a]&&(n=!1)}n&&(u.splice(r--,1),e=p(p.s=t[0]))}return e}var n={},o={1:0},u=[];function p(r){if(n[r])return n[r].exports;var t=n[r]={i:r,l:!1,exports:{}};return e[r].call(t.exports,t,t.exports,p),t.l=!0,t.exports}p.m=e,p.c=n,p.d=function(e,r,t){p.o(e,r)||Object.defineProperty(e,r,{enumerable:!0,get:t})},p.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},p.t=function(e,r){if(1&r&&(e=p(e)),8&r)return e;if(4&r&&"object"==typeof e&&e&&e.__esModule)return e;var t=Object.create(null);if(p.r(t),Object.defineProperty(t,"default",{enumerable:!0,value:e}),2&r&&"string"!=typeof e)for(var n in e)p.d(t,n,function(r){return e[r]}.bind(null,n));return t},p.n=function(e){var r=e&&e.__esModule?function(){return e.default}:function(){return e};return p.d(r,"a",r),r},p.o=function(e,r){return Object.prototype.hasOwnProperty.call(e,r)},p.p="./";var l=this.webpackJsonpdscwebapp=this.webpackJsonpdscwebapp||[],a=l.push.bind(l);l.push=r,l=l.slice();for(var f=0;f<l.length;f++)r(l[f]);var i=a;t()}([])</script><script src="./static/js/2.a3d51a66.chunk.js"></script><script src="./static/js/main.6725a856.chunk.js"></script></body></html>
```

Comment:
What happened to this?

Reply:
just build changes, doesnt change the actual code